controllers/admin/AdminGroupsController.php

```diff
@@ -232,6 +259,7 @@ protected function renderCustomersList($group)
         $this->bulk_actions = false;
         $this->list_no_link = true;
         $this->explicitSelect = true;
+        $this->list_skip_actions = [];
```

Comment:
Why not deleting this line instead: https://github.com/PrestaShop/PrestaShop/pull/25295/files#diff-a21493578f308609b40c68896f45a7988d38a0fbb598f7d3f96765fdadd190e3R92 ?

Reply:
@atomiix It's because the page is used for listing groups and customers of each group. I have tested the same deletion in first try :tongue: